### PR TITLE
#166661313 Refactor Update device mutation

### PR DESCRIPTION
**Pull request by vic3king**
#### What does this PR do?
Refactor implementation for updating a device
#### Description of Task to be completed?
Currently, when an admin tries to update device information, the `date_added` and `last_seen` fields are updated as well. This behaviour is not what is expected. This PR fixes that issue.
#### How should this be manually tested?
1. Clone the repo: `git clone https://github.com/andela/mrm_api.git`
2. Setup project according to Readme.md
3. checkout to branch `ch-refactor-update-device-mutation-166661313`
4. When using Docker `make migrate`

#### What are the relevant pivotal tracker stories?
[#166661313](https://www.pivotaltracker.com/story/show/166661313)

#### Background context
N/A

Screenshots
N/A

- [x]  My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [x] I have made the corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Implementation works according to expectations

### CODE REVIEW

File: api/devices/schema.py [hunk 1]
```diff
@@ -78,8 +78,6 @@ def mutate(self, info, device_id, **kwargs):
         ).first()
         if not exact_device:
             raise GraphQLError("DeviceId not found")
-        exact_device.date_added = datetime.now()
```
Comment (codinger41) [inline api/devices/schema.py]: @vic3king Great job, however, do you think we should update the `date_added` field?

Comment (vic3king) [reply api/devices/schema.py]: @codinger41 I removed that line. we are no longer updating the `date-added`

Comment (codinger41) [reply api/devices/schema.py]: Oh sorry, great job!

File: api/devices/schema.py [hunk 2]
```diff
@@ -78,8 +78,6 @@ def mutate(self, info, device_id, **kwargs):
         ).first()
         if not exact_device:
             raise GraphQLError("DeviceId not found")
```
Comment (Arusey) [inline api/devices/schema.py]: Kindly add a space between the Device and ID message. For aesthetic purposes

Comment (vic3king) [reply api/devices/schema.py]: I'll fix that.👍🏼

**Comment by vic3king** [comment]
> I noticed you are using `make ssh service=app` when trying to run migrations on docker. A simpler way would just run `make migrate` or run `make migrate message="migration message"` when generating a new file in docker. Kindly note that making a secure shell is only useful when migration commands are not working as expected.

thanks for pointing that out.

**Pull request merged**